### DISCUSSION TITLE
Install from Open WebUI Github Repo, no login interface appears

### DISCUSSION THREAD

**Discussion by hito0512** [open-webui/open-webui#2569] [General]
![1](https://github.com/open-webui/open-webui/assets/26110883/4cb9a820-385a-48b7-81dd-59812d3d941f)

No login interface appears,Stuck here all the time

build with python
![2](https://github.com/open-webui/open-webui/assets/26110883/31234ce5-421f-43ee-9b3c-fc2fd08cc186)

**Comment by justinh-rahb** [comment]
https://docs.openwebui.com/getting-started/#install-from-open-webui-github-repo

**Comment by ghostshield-21** [comment]
i got the same issue after installation using 
`pip install open-webui`

log as following
![image](https://github.com/open-webui/open-webui/assets/3272028/3be38ae7-6d1c-479e-99b7-630923850dd8)
